DataConnectors/FORCEPOINT_NGFW.json

```diff
@@ -0,0 +1,116 @@
+{
+  "id": "FORCEPOINT_NGFW",
+  "title": "Forcepoint NGFW",
+  "publisher": "Forcepoint",
+  "descriptionMarkdown": "The Forcepoint NGFW connector allows you to automatically export user-defined Forcepoint NGFW logs into Azure Sentinel in real-time. This enriches visibility into user activities recorded by NGFW, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.​",
```

Comment:
1. Expand out the abbreviation at least one time initially as "the Forcepoint NGFW (Next Generation Firewall) connector allows..."
2. I see formatting issues when I load this connector at the end of the description 'Azure Sentinel.' Just delete some part of 'Sentinel.' and the retype it to remove the weird characters and import the file to validate using the import test tool/functionality.

Reply:
Updated, thanks

Reply:
Thanks, for resolving #1 - #2 is not resolved